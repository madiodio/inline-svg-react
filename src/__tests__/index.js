import React from 'react';
import renderer from 'react-test-renderer';

import InlineSVG from '..';

describe('Component: InlineSVG', () => {
  const Icon = '<svg viewBox="0 0 21 21"><path d="M13 23h7V8L10 .631 0 8v15h7v-7h6v7z"/></svg>';
  it('should match content with specified props', () => {
    const tree = renderer
      .create(<InlineSVG icon={Icon} size={20} label="home - house" isTest />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
