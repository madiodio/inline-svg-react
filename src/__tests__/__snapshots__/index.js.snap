// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: InlineSVG should match content with specified props 1`] = `
<span
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg role=\\"img\\" aria-labelledby=\\"id0\\" width=\\"20\\" height=\\"20\\"  viewBox=\\"0 0 21 21\\"><title id=id0>Icon - home - house</title><path d=\\"M13 23h7V8L10 .631 0 8v15h7v-7h6v7z\\"/></svg>",
    }
  }
/>
`;
